tests/admin/views/test_emergency_shutoff.py

```diff
@@ -38,19 +38,23 @@ def test_get_by_id(self):
         self.assertEqual(data["product"], "Fennec")
         self.assertIn("channel", data)
         self.assertEqual(data["channel"], "beta")
+        self.assertIn("comment", data)
+        self.assertEqual(data["comment"], "Fennec panic!()")
 
     def test_get_notfound(self):
         resp = self._get("/emergency_shutoff/foo/bar")
         self.assertStatusCode(resp, 404)
 
     def test_create(self):
-        data = {"product": "Thunderbird", "channel": "release"}
+        data = {"product": "Thunderbird", "channel": "release", "comment": "Thunderbird panic!()"}
```

Comment:
We should keep a test for POST to /emergency_shutoff without a comment, to make sure that still works.